ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -153,15 +153,21 @@ extension BrowserViewController {
 
 extension Tab {
   func reportPageLoad(to rewards: BraveRewards, redirectChain urls: [URL]) {
-    guard let webView = webView, let url = webView.url else { return }
-
-    if url.isLocal || self.isPrivate { return }
+    guard let url = urls.last, let webView = webView, !url.isLocal, !isPrivate
```

Comment:
nit: use `redirect_chain` instead of `urls`, i.e., `redirect_chain = redirect_chain.last`

Reply:
Fixed